src/client/activity/UserActivityActionsList.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import {
+  getUser,
+  getAuthenticatedUser,
+  getAuthenticatedUserName,
+  getTotalVestingShares,
+  getTotalVestingFundSteem,
+  getUsersAccountHistory,
+  getLoadingMoreUsersAccountHistory,
+  getUserHasMoreAccountHistory,
+  getAccountHistoryFilter,
+  getCurrentDisplayedActions,
+  getCurrentFilteredActions,
+} from '../reducers';
+import { isWalletTransaction } from '../helpers/apiHelpers';
+import {
+  setInitialCurrentDisplayedActions,
+  addMoreActionsToCurrentDisplayedActions,
+  loadMoreCurrentUsersActions,
+} from '../wallet/walletActions';
+import ReduxInfiniteScroll from '../vendor/ReduxInfiniteScroll';
+import WalletTransaction from '../wallet/WalletTransaction';
+import UserAction from './UserAction';
+
+@withRouter
+@connect(
+  (state, ownProps) => ({
+    user: ownProps.isCurrentUser
+      ? getAuthenticatedUser(state)
+      : getUser(state, ownProps.match.params.name),
+    totalVestingShares: getTotalVestingShares(state),
+    totalVestingFundSteem: getTotalVestingFundSteem(state),
+    usersAccountHistory: getUsersAccountHistory(state),
+    loadingMoreUsersAccountHistory: getLoadingMoreUsersAccountHistory(state),
+    userHasMoreActions: getUserHasMoreAccountHistory(
+      state,
+      ownProps.isCurrentUser
+        ? getAuthenticatedUserName(state)
+        : getUser(state, ownProps.match.params.name).name,
+    ),
+    accountHistoryFilter: getAccountHistoryFilter(state),
+    currentDisplayedActions: getCurrentDisplayedActions(state),
+    currentFilteredActions: getCurrentFilteredActions(state),
+  }),
+  {
+    setInitialCurrentDisplayedActions,
+    addMoreActionsToCurrentDisplayedActions,
+    loadMoreCurrentUsersActions,
+  },
+)
+class UserActivityActionsList extends Component {
+  static propTypes = {
+    currentDisplayedActions: PropTypes.arrayOf(PropTypes.shape()),
+    currentFilteredActions: PropTypes.arrayOf(PropTypes.shape()),
+    accountHistoryFilter: PropTypes.arrayOf(PropTypes.string),
+    userHasMoreActions: PropTypes.bool.isRequired,
+    loadingMoreUsersAccountHistory: PropTypes.bool.isRequired,
+    isCurrentUser: PropTypes.bool, // eslint-disable-line react/no-unused-prop-types
```

Comment:
Why is it there if not used?

Reply:
used in lines 32 and 42, eslint doesn't recognize it :(